config/default.js

```diff
@@ -312,4 +313,10 @@ module.exports = {
   enableAMInstallButton: false,
 
   enableInlineAddonReview: false,
+
+  // The withExperiment HOC relies on this config to enable/disable A/B
+  // experiments.
+  experiments: {
```

Comment:
should we have this match the hoc name: `withExperiments: { .. }`

Reply:
mmh, I think it is shorter to use `experiments` and to have a list of experiments here. The name `withExperiment` is an implementation detail to me.

Reply:
ok I'll defer to you : )